katago_server/trainings/viewsets/network.py

```diff
@@ -15,3 +15,6 @@ class NetworkViewSet(viewsets.ModelViewSet):
     queryset = Network.objects.all()
     permission_classes = [IsAdminUser | ReadOnly]
     serializer_class = NetworkSerializer
+    # TODO how to get this to work??
+    # lookup_field = "name"
```

Comment:
What was the problem to get it work ?

Reply:
Solved, the problem was my local server still had a bunch of junk fake data with non unique and with blank names, even though the field is now required to be unique.